plugin/supervisor.go

```diff
@@ -4,7 +4,10 @@
 package plugin
 
 import (
+	"crypto/md5"
```

Comment:
Why was `md5` picked here over other hash algorithms?  Does it matter that md5 is cryptographically broken?

Reply:
I don't mind using another.

Md5 is less costly in terms of performance and an attacker would need to generate a binary of their *evil* code that could generate the same hash as the plugin's executable file.

Reply:
switched to sha256, thanks Ben!